sdk/identity/azure-identity/pom.xml

```diff
@@ -81,5 +81,11 @@
       <version>3.3.0.RELEASE</version> <!-- {x-version-update;io.projectreactor:reactor-test;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
```

Comment:
1.2.0 is out for this library.
Is that what we intend to add here ?

Reply:
Using 1.2.0 now